fedimint-client/src/lib.rs

```diff
@@ -1080,8 +1080,8 @@ impl Client {
                 "We only build transactions with input versions that are supported by the module",
             );
 
-            in_amount += input.amount;
-            fee_amount += item_fee;
+            in_amount.saturating_add_to(input.amount);
+            fee_amount.saturating_add_to(item_fee);
```

Comment:
Saturating is incorrect here imo. We can just make the fn fallible. If we ever overflow the tx will be rejected by the federation anyway.